common/app/views/fragments/nav/userAccountDropdown.scala.html

```diff
@@ -26,11 +26,14 @@
         My account
     </label>
 
-    <ul class="dropdown-menu js-user-account-dropdown-menu"
+    <ul class="dropdown-menu dropdown-menu--light js-user-account-dropdown-menu"
     id="my-account-dropdown"
     aria-hidden="true">
 
     @for((item) <- accountDropdownMenu) {
+        @if(item.divider) {
+            <hr/>
```

Comment:
What is this, XML? 🙀

Reply:
haha I'll tell myself if I leave `<hr>` open html5 style I'd be a hypocrite for using `</li>`